README.md

```diff
@@ -641,6 +641,23 @@ it requires completely separate logic from `foreach/2` and `reduce`
 in both the parser and the interpreter.
 
 
+## Error handling
+
+In jq, the `try f catch g` expression breaks out of the `f` stream as
+soon as an error occurs, ceding control to `g` after that. This is
+mentioned in its manual as a possible mechanism for breaking out of
+loops
+([here](https://jqlang.github.io/jq/manual/#breaking-out-of-control-structures)). jaq
+however doesn't interrupt the `f` stream, but instead sends _each_
+error value emitted to the `g` filter; the result is a stream of
+values emitted from `f` with values emitted from `g` interspersed
+where errors ocurred.
+
+```jq
+[try (1, error(2), 3, error(4)) catch .] == [1, 2, 3, 4]
+```
```

Comment:
I would clarify that this output is from jaq, and I would also include the output of jq for the same example and mark it accordingly.